src/daemon/daemon.h

```diff
@@ -41,6 +42,8 @@ class Daemon : public QObject {
   QString logs();
   void cleanLogs();
 
+  DaemonSession* session() { return &m_session; }
```

Comment:
Should this move into DaemonLocalServer?

Reply:
I put it here mostly for convenience. The connection has access to the Daemon already and not to the DaemonLocalServer.

Reply:
Putting it in Daemon exposes it to the Linux, which doesn't use it (and requires the linux.cmake change) . A potential solution may be to add it to DaemonLocalServer as follows (either return a reference or pointer to the singleton):

```
DaemonAccessControl& DaemonLocalServer::accessControl() {
  static DaemonAccessControl s_accessControl;
  return s_accessControl;
}
```

Then the caller can get it as :

```
  auto accessControl = DaemonLocalServer::accessControl();
  accessControl.resetSession();

```

Reply:
Woops, forgot to do this one.

Reply:
Done.